native/templates/ios/View.mm

```diff
@@ -24,16 +24,15 @@ of this software and associated engine source code (the "Software"), a limited,
 ****************************************************************************/
 
 #import "View.h"
-#import "platform/ios/AppDelegate.h"
```

Comment:
View.mm don't have to be in template. It can be inside engine, as it is added by engine. It is not a concept of iOS. And it is related to iOS rendering, the logic may change.

Reply:
ok, i'll change it